workitem/link/link_repository.go

```diff
@@ -88,16 +96,28 @@ func (r *GormWorkItemLinkRepository) ValidateCorrectSourceAndTargetType(ctx cont
 		return errs.WithStack(err)
 	}
 	// Check type paths
```

Comment:
why are you talking about "type paths" here ? isn't it about "type hierarchy" instead ? From my understanding, a link type combination is valid if the source and target link types match the ones described in the combination or if they are subtypes of the ones described in the combination. Correct ?